.github/workflows/pre-submit.actions.yml

```diff
@@ -498,17 +499,14 @@ jobs:
         uses: ./.github/actions/detect-workflow-js
       - uses: ./.github/actions/generate-builder
         with:
-          repository: ${{ steps.detect.output.repository }}
+          repository: ${{ steps.detect.outputs.repository }}
```

Comment:
good catch!

Reply:
Yeah, these are the kinds of things that the `actiontest` tool can find so it's pretty useful.